packages/web3-utils/index.d.ts

```diff
@@ -0,0 +1,137 @@
+import BigNumber from "bn.js";
+import us from "underscore";
```

Comment:
I moved from the underscore package to the underscore-es package otherwise I can't import just single methods :)

Reply:
if that's the case we can probably remove this completely from the types now.